tests/sentry/event_manager/test_event_manager.py

```diff
@@ -1049,19 +1049,6 @@ def test_throws_when_matches_discarded_hash(self):
             assert o.kwargs["category"] == DataCategory.ATTACHMENT
             assert o.kwargs["quantity"] == 5
 
-        def query(model, key, **kwargs):
-            return tsdb.get_sums(model, [key], event.datetime, event.datetime, **kwargs)[key]
-
-        # Ensure that we incremented TSDB counts
-        assert query(tsdb.models.organization_total_received, event.project.organization.id) == 2
-        assert query(tsdb.models.project_total_received, event.project.id) == 2
-
-        assert query(tsdb.models.project, event.project.id) == 1
-        assert query(tsdb.models.group, event.group.id) == 1
-
-        assert query(tsdb.models.organization_total_blacklisted, event.project.organization.id) == 1
-        assert query(tsdb.models.project_total_blacklisted, event.project.id) == 1
```

Comment:
This test is actually pretty important for org stats. Are we sure that this is properly tested for the other backend?

Reply:
The snuba backend is quite well-tested: https://github.com/getsentry/sentry/blob/ref/sentry-inbound-filters/tests/sentry/tsdb/test_snuba.py

As long as we make sure `track_outcome` is called in EventManager, and make sure outcomes result in tsdb incrs (which we do in the snuba tsdb testsuite), I think we should be good.

Reply:
Alright, considering that outcomes are checked just above this, let's go ahead with this 👍